curriculum/challenges/english/25-front-end-development/lecture-working-with-css-transforms-overflow-and-filters/672bcc9c4a6dd6d7dd3e6357.md

```diff
@@ -8,7 +8,51 @@ dashedName: what-is-the-css-box-model
 
 # --description--
 
-Watch the lecture video and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
+What is the CSS box model, and how does it work?
```

Comment:
We have this issue here about there being an error in the video.

https://github.com/freeCodeCamp/freeCodeCamp/issues/58830

Can you double check the script and see if it also needs to be updated?